### ISSUE TITLE
Clarify some aspects of the specifications

### ISSUE THREAD

**Issue by jonrkarr** [SED-ML/sed-ml#180]
- [ ] `new entities might be specified using \hyperref[sec:mathML]{MathML}` at https://github.com/SED-ML/sed-ml/blob/master/specification/level-1-version-4/sources/introduction.tex#L48
- [ ] 2.1.1.6 : `but does not imply accessibility of that variable`
- [ ] 2.1.2: `In order to set either Notes or Annotation on a SED-ML class the metaid is required.` Is this really required? RDF can treat this nodes without metaids as blank nodes.
- [ ] 2.1.6: `A Variable (Figure 2.5) is a reference to an already existing entity, either explicitly created in the SED-ML Document, or to an implicitly defined symbol.` This statement is hard to understand. This should be more clearly say that variables are typically references to entities in the specifications of models/state of simulations. 
- [ ] 2.2.8.1 `but in parallel tasks, a Model is to assume to its initial state.` I find this wording hard to parse. I think the intended meaning is that setValue changes within a repeated task should apply only to that task (and not other tasks). This is unrelated to parallel execution.
- [ ] 2.2.8.2 `generic looping construct`. I think this could be confusing. This could be construed to mean RepeatedTask is similar to for or while loop in a programming language. Rather, RepeatedTask enables a very specific form of iteration.

**Comment by luciansmith** [comment]
Updated:

'new entities...' -> 
"In the definition of a DataGenerator, any addressable variable or parameter of any model or DataSource may be referenced, and used to calculate new values using MathML.

'but does not imply' -> 
"A URN is a colon-separated string that references an external variable.  It may or may not be accessible online, but should have a unique definition."

'metaid is required'  This was already changed from what you quote; maybe you changed it in your other readthrough.  It now reads "A metaid is required to apply a Notes or Annotation to a SED-ML element." which I think is clearer.

'A Variable...' ->
"A Variable (Figure 2.5) is a reference to the mathematical value of an explicit model element such as
an SBML Species or a CellML Variable, the mathematics implicitly defined by a model such as the
stoichiometry matrix, or a DataGenerator in this same SED-ML document."

'but in parallel' ->
"Inside a \RepeatedTask, the state of the model may have been changed, \changed{otherwise, the} \Model is to assume to its initial state."
[Replaced the parallel bit with 'otherwise', since that's what it was supposed to be implying.)

'generic looping construct' ->
"The RepeatedTask (Figure 2.13 on the next page) provides a looping construct, allowing complex tasks
to be composed from individual tasks."
